lib/licensee/project_files/license_file.rb

```diff
@@ -34,11 +34,12 @@ class LicenseFile < Licensee::ProjectFiles::ProjectFile
         /#{COPYING_REGEX}-/                          => 0.65,  # COPYING-MIT
         /-#{LICENSE_REGEX}/                          => 0.60,  # MIT-LICENSE-MIT
         /-#{COPYING_REGEX}/                          => 0.55,  # MIT-COPYING
-        /\A#{OFL_REGEX}#{PREFERRED_EXT_REGEX}/       => 0.50,  # OFL.md
-        /\A#{OFL_REGEX}#{ANY_EXT_REGEX}/             => 0.45,  # OFL.textile
-        /\A#{OFL_REGEX}\z/                           => 0.40,  # OFL
-        /\A#{PATENTS_REGEX}\z/                       => 0.35,  # PATENTS
-        /\A#{PATENTS_REGEX}#{ANY_EXT_REGEX}\z/       => 0.30,  # PATENTS.txt
+        /#{LICENSE_REGEX}_/                          => 0.50,  # LICENSE_1_0.txt
```

Comment:
Another approach would be to just accept _ wherever - is. I don't think filename scoring is important now that we support multiple license files (but I could be missing something), so a new score for a new pattern isn't important.

Another approach would be to add a match for LICENSE_1_0.txt ie exactly the filename mentioned in #221.

Reply:
Very true. We still use it within GitHub.com. Even if the license is ultimately deemed `other`, we still link to the preferred license file. This could potentially be a breaking change we queue up for the next major version.

Reply:
Does GitHub.com link to a preferred license file even if `other` is detected? I can't think of where.